web-app/src/contexts/dictonary.js

```diff
@@ -0,0 +1,23 @@
+import PropTypes from "prop-types";
+import { createContext } from "react";
+import { useQuery } from "react-query";
+
+export const DictonaryContext = createContext();
```

Comment:
The contexts can be combied as one later if needed as there is not much data in each.